app/src/main/java/org/fossasia/openevent/Adapters/TracksListAdapter.java

```diff
@@ -42,15 +53,31 @@ public int getItemCount() {
         return tracks.size();
     }
 
+    private void setAnimation(View viewToAnimate, int position) {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
why the version comparison? Also `lastPosition` should be reset when the Animation has ended that's the idea right?